requirements.txt

```diff
@@ -1,7 +1,27 @@
-django==4.1
```

Comment:
Do not include the `media/uploads/movies` folder with it's content to the pull request.